tls/s2n_kem.h

```diff
@@ -140,3 +151,21 @@ int s2n_kem_recv_ciphertext(struct s2n_stuffer *in, struct s2n_kem_params *kem_p
 int s2n_kyber_512_r3_crypto_kem_keypair(OUT uint8_t *pk, OUT uint8_t *sk);
 int s2n_kyber_512_r3_crypto_kem_enc(OUT uint8_t *ct, OUT uint8_t *ss, IN const uint8_t *pk);
 int s2n_kyber_512_r3_crypto_kem_dec(OUT uint8_t *ss, IN const uint8_t *ct, IN const uint8_t *sk);
+
+/* kyber768r3 */
+#define S2N_KYBER_768_R3_PUBLIC_KEY_BYTES    1184
+#define S2N_KYBER_768_R3_SECRET_KEY_BYTES    2400
+#define S2N_KYBER_768_R3_CIPHERTEXT_BYTES    1088
+#define S2N_KYBER_768_R3_SHARED_SECRET_BYTES 32
+int s2n_kyber_768_r3_crypto_kem_keypair(OUT uint8_t *pk, OUT uint8_t *sk);
+int s2n_kyber_768_r3_crypto_kem_enc(OUT uint8_t *ct, OUT uint8_t *ss, IN const uint8_t *pk);
+int s2n_kyber_768_r3_crypto_kem_dec(OUT uint8_t *ss, IN const uint8_t *ct, IN const uint8_t *sk);
```

Comment:
Where are these methods (and the 1024 versions) implemented? I'm not finding them with a code search

Reply:
ah, i believe these are vestigial from my first cut at the implementation. thanks for catching. their 512 counterpart is used to wrap s2n's interned kyber512 implementation, and are unnecessary for 768 and 1024.

removed.